docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
   mysql:
     image: mysql:8
     ports:
-      - "3306"
+      - "3307:3306"
```

Comment:
Hum, I'm not sure why you have to switch this port. Can you explain?

Reply:
Of course :smile: 

If you specify only the container port, an ephemeral port will be created for the host. 
This is currently the case.

I propose to specify the port of the host, here 3307. This avoids going to look for the port chosen by docker.

See https://docs.docker.com/compose/compose-file/compose-file-v3/#ports for more informations

Reply:
Forcing a port is not good (I have already something listening on this port for example), at least this should be configurable, within the override file.

Reply:
On the prestashop-git service the ports are forced to 8001 and 8002, I could also have these ports used on my host. Might as well do it on the override file

Reply:
I guess this can be configurable as well, so maybe moving it to the override file can be an idea.

Reply:
I pushed a proposal in this direction